code/game/objects/items/binoculars.dm

```diff
@@ -200,12 +203,9 @@
 		if(MODE_RANGE_FINDER)
 			var/obj/effect/overlay/temp/laser_coordinate/LT = new (TU, laz_name, S)
 			coord = LT
-			to_chat(user, "<span class='notice'>SIMPLIFIED COORDINATES OF TARGET. LONGITUDE [coord.x]. LATITUDE [coord.y].</span>")
+			to_chat(user, "<span class='notice'>COORDINATES: LONGITUDE [coord.x]. LATITUDE [coord.y].</span>")
```

Comment:
Might as well just not even create the `obj/effect/overlay/temp/laser_coordinate`.
Just get the position of `TU` instead